### DISCUSSION TITLE
網頁後端常用語言和框架

### DISCUSSION THREAD

**Discussion by minniewei** [1101-GS4719-C/Python-Community-GS4719-C#627] [General]
https://medium.com/johnliu-%E7%9A%84%E8%BB%9F%E9%AB%94%E5%B7%A5%E7%A8%8B%E6%80%9D%E7%B6%AD/%E5%BE%8C%E7%AB%AF%E5%B7%A5%E7%A8%8B%E5%B8%AB%E7%9A%84%E7%AC%AC%E4%B8%80%E5%A0%82%E8%AA%B2-1-%E8%AA%9E%E8%A8%80%E7%9A%84%E9%81%B8%E6%93%87%E8%88%87%E6%AF%94%E8%BC%83-%E5%8B%95%E6%85%8B%E8%AA%9E%E8%A8%80-1ce90f8ddd32

透過這篇文章我們可以了解到現今火紅的後端語言和框架。我覺得慎選框架是一件很重要的事喔喔喔喔！！！（強烈推薦javascript(個人私心...咳)）

**Comment by gnsJhenJie** [comment]
學校有不少系統都是用Laravel開發的，我自己用過也覺得蠻好用的，不過官網的說明文件我覺得寫的不是很詳細，還蠻常有要去翻API Docs的狀況。
